src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceModelHttpMessageHandler.NetNative.cs

```diff
@@ -79,11 +79,6 @@ public bool SupportsCertificateValidationCallback
         {
             get { return false; }
         }
-
-        protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
-        {
-            return base.SendAsync(request, cancellationToken);
-        }
```

Comment:
These are partial classes and not derived types. So the SendAsync method in NetNative was a duplicate method with the one in the base type. The implementation in NetNative was a harmless leftover from when we had a separate wrapper around WinHttpHandler for windows coreclr. Now it's no longer harmless and was causing a problem.